src/types/error_parser.ts

```diff
@@ -0,0 +1,67 @@
+import * as _ from "lodash";
+
+import { Logging } from "./logging";
+import { DebtKernelError } from "./debt_kernel_error";
+import { RepaymentRouterError } from "./repayment_router_error";
+
+export interface ContractAddressBook {
```

Comment:
I like this nomenclature -- we should use it elsewhere.  No requested change here, but we can discuss this in the future.

Reply:
I like it too. Thought it was clever ;) Glad to know I wasn't the only one.